### ISSUE TITLE
jax_api_example calibrates Gaussian noise for the clipped sum then adds it to the mean

### ISSUE THREAD

**Issue by Anu27n** [google-deepmind/jax_privacy#383]
`examples/jax_api_example.py` sets

`
stddev = noise_multiplier * clipped_grad.sensitivity(REPLACE_ONE)
`

`clipped_grad` (default `normalize_by=1`) returns a **sum** of clipped per-example gradients, so that sensitivity is for the sum. The training step then does

`
mean_grads = grads / batch_size
privatizer.update(mean_grads, ...)
`

Noise calibrated for the sum is applied to the mean, so the example over-noises by about `batch_size`. Other examples (`dp_logistic_regression.py`) privatize the clipped sum, then divide.

The examples guide already flags this file as research/demo (fixed-size batches, not Poisson). The scale mismatch is a separate correctness bug in the DP path.
